fetch/nosniff/resources/nosniff.py

```diff
@@ -1,10 +1,12 @@
+from wptserve.utils import isomorphic_decode
```

Comment:
Looks like `isomorphic_decode` is imported but never used here.

Reply:
Updated. Thank you!